src/fragments/lib/auth/js/switch-auth.mdx

```diff
@@ -74,13 +74,47 @@ Auth.signIn(username, password)
   .catch(err => console.log(err));
 ```
 
+<Callout>
+
+When using Custom Auth Flows, you can setup a PasswordFull or Passwordless Sign In. This decision will determine how your Custom Auth Lambda functions are defined.
```

Comment:
Should it be `Passwordfull` or `PasswordLess`?

Reply:
I think we should should say Password or Passwordless - Passwordful sounds awkward to me